### ISSUE TITLE
Microsoft (R) Test Execution CLI, Copyright (C) Microsoft Corporation. All rights reserved.

### ISSUE THREAD

**Issue by tmds** [microsoft/vstest#4869]
On a source-build SDK, the `vstest.console.dll` (which comes as part of the distro SDK) includes these lines when you run `dotnet test`:

```
Microsoft (R) Test Execution Command Line Tool Version 17.8.0 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.
```

For MSBuild, we removed the Microsoft trademark/copyright bits in https://github.com/dotnet/msbuild/pull/7672 and changed
```
Microsoft (R) Build Engine version 17.0.0+c9eb9dd64 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.
```
into:
```
MSBuild version 17.8.3+195e7f5a3 for .NET
```

Can we make a similar change to the vstest cli?

cc @richlander @baronfel @omajid

**Comment by nohwnd** [comment]
Sure, we can do it if @richlander says it is okay. I don't have any details about what needed to be discussed or approved for this to happen in MSBuild. 

I searched [grep.app](https://grep.app/search?q=Test%20Execution%20Command%20Line%20Tool) and we will most likely break dotnet/sdk, and few other people testing our output, but that happens every time we touch our output. 

https://github.com/dotnet/sdk/blob/main/test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs


Will check the naming with our team: `VSTest version 17.8.0 (x64)`

**Comment by tmds** [comment]
> Sure, we can do it if @richlander says it is okay.

@richlander can you take a look?

**Comment by tmds** [comment]
@richlander, friendly reminder.

**Comment by richlander** [comment]
Wow. This one skipped by me. Please accept my apologies!

LGTM

**Comment by MichaelSimons** [comment]
Any progress on a fix for this?

**Comment by nohwnd** [comment]
Thanks for the reminder, implemented it in the linked PR, will be released in 17.11.
